Dockerfile

```diff
@@ -82,8 +82,8 @@ RUN mkdir -p ${APP_ROOT} && \
     curl -L https://github.com/ManageIQ/manageiq-ui-self_service/tarball/${REF} | tar vxz -C ${SSUI_ROOT} --strip 1
 
 ## Add ManageIQ source from local directory (dockerfile development) or from Github (official build)
-# ADD . ${APP_ROOT}
-RUN curl -L https://github.com/ManageIQ/manageiq/tarball/${REF} | tar vxz -C ${APP_ROOT} --strip 1
+ADD . ${APP_ROOT}
```

Comment:
@bazulay My thought is that per Docker's "Dockerfile Best Practices" doc under the "ADD or COPY" section (https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/):

> Although ADD and COPY are functionally similar, generally speaking, COPY is preferred. That’s because it’s more transparent than ADD. COPY only supports the basic copying of local files into the container, while ADD has some features (like local-only tar extraction and remote URL support) that are not immediately obvious. Consequently, the best use for ADD is local tar file auto-extraction into the image, as in ADD rootfs.tar.xz /.

Overall, in this single instance / use case, I agree with your assessment it will most likely not change the image size, however moving forward I wonder if we want to standardize on COPY to help ensure we don't get that (apologies if I was unclear).

So to your point about "if we change it than it should be for the right reason", I feel that doing what is in the vendors best practices guide is the right reason.